src/Sylius/Behat/Page/Admin/Shipment/IndexPage.php

```diff
@@ -43,6 +43,15 @@ private function getField(string $orderNumber, string $fieldName): NodeElement
         return $field = $tableAccessor->getFieldFromRow($table, $row, $fieldName);
     }
 
+    public function showOrderPageRelatedWithShipment(string $orderNumber): void
+    {
+        $tableAccessor = $this->getTableAccessor();
+        $table = $this->getElement('table');
+        $row = $tableAccessor->getRowWithFields($table, ['number' =>$orderNumber]);
+        $field = $tableAccessor->getFieldFromRow($table, $row, 'actions');
+        $field->clickLink('Details');
```

Comment:
I would extract

```php
$tableAccessor = $this->getTableAccessor();
$table = $this->getElement('table');
$row = $tableAccessor->getRowWithFields($table, ['number' =>$orderNumber]);
$field = $tableAccessor->getFieldFromRow($table, $row, 'actions');
```

to some private function like `getActionsForRow(array $parameters): NodeElement`, to make it more readable :)

Reply:
And missing space before `$orderNumber`